### ISSUE TITLE
 cls_accuracy:0.0000

### ISSUE THREAD

**Issue by lijain** [tristandb/EfficientDet-PyTorch#10]
Thanks for the work of sharing, after calling your bifpn on retinanet cls_accuracy:0.0000.
 At first, I thought it was caused by the lack of alignment of my texture, and finally changing the texture to 32 and other divisible ones would cause this problem; then Baidu adjusted the learning rate or not. What is the reason for this?
